config/config/wc-config.yaml

```diff
@@ -22,53 +27,95 @@ storageClasses:
   ebs:
     enabled: null # true | false
 
+  storageClass: "set-me"
+
+  ## IP of the cluster DNS in kubernetes
+  ##
+  clusterDns: "10.43.0.10"
+
+## Object storage configuration for backups.
+##
 objectStorage:
-  # Options are "s3", "gcs", or "none"
-  # If "none", remember to disable backups (velero)
+  ## Options are "s3", "gcs", or "none"
+  ## If "none", remember to disable backups (velero)
   type: "s3"
   # gcs:
   #   project: "set-me"
   # s3:
   #   region: "set-me"
   #   regionAddress: "set-me"
   #   regionEndpoint: "set-me"
+
+  ##
+  ## TODO: remove all but for 'velero'.
+  ##
   buckets:
     harbor: "${CK8S_ENVIRONMENT_NAME}-harbor"
     velero: "${CK8S_ENVIRONMENT_NAME}-velero"
     elasticsearch: "${CK8S_ENVIRONMENT_NAME}-es-backup"
     influxDB: "${CK8S_ENVIRONMENT_NAME}-influxdb"
     scFluentd: "${CK8S_ENVIRONMENT_NAME}-sc-logs"
 
+## User configuration.
 user:
   # This only controls if the namespaces should be created, user RBAC is always created.
   createNamespaces: true
+  ## List of user namespaces to create.
+  ##
   namespaces:
```

Comment:
Should we make sure to leave a empty line above all these comments? (seems like that how you've done it for most of the others